pipelines/template-build/template-build.yaml

```diff
@@ -162,6 +162,19 @@ spec:
         value: $(tasks.build-container.results.IMAGE_DIGEST)
       - name: image-url
         value: $(tasks.build-container.results.IMAGE_URL)
+    - name: preflight
+      when:
+      - input: $(params.skip-checks)
+        operator: in
+        values: ["false"]
+      runAfter:
+        - build-container
+      taskRef:
+        name: preflight
```

Comment:
Do we have to reference the task using name `ecosystem-cert-preflight-checks` here?

Reply:
Thanks for pointing it out.. Totally missed that!